### PR TITLE
sql: Make the description of crdb_internal.set_vmodule() useful

### PR DESCRIPTION
**Pull request by andreimatei**
Release note: None

### CODE REVIEW

**Review by andreimatei — COMMENTED** [review]
friendly ping

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/29359#-:-LVsPLDjCL1IVmWCl2Qk:b-50swta)* status: :shipit: complete! 0 of 0 LGTMs obtained

**Comment by andreimatei** [comment]
@petermattis , Tobi's not giving me the love

File: pkg/sql/sem/builtins/builtins.go [hunk 1]
```diff
@@ -2873,8 +2873,11 @@ may increase either contention or retry errors, or both.`,
 			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
 				return tree.DZero, log.SetVModule(string(*args[0].(*tree.DString)))
 			},
-			Info: "This function is used for internal debugging purposes. " +
-				"Incorrect use can severely impact performance.",
+			Info: "Set the equivalent of the --vmodule flag on the gateway node processing this request; " +
+				"it affords control the logging verbosity of different files. " +
```
Comment (jordanlewis) [inline pkg/sql/sem/builtins/builtins.go]: nit: "affords control the" doesn't make too much sense to me. Can you change this?

**Review by andreimatei — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/29359)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @petermattis and @tbg)

___
*[pkg/sql/sem/builtins/builtins.go, line 2877 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/29359#-LLAtaDfOAhpaID-zjhX-r1-2877:-LXQW2pv7AlJDIcEJyJE:bb7f2mf) ([raw file](https://github.com/cockroachdb/cockroach/blob/a23c39f41ed10a38fb49d767e18cc686396448cc/pkg/sql/sem/builtins/builtins.go#L2877)):*
<details><summary><i>Previously, jordanlewis (Jordan Lewis) wrote…</i></summary><blockquote>

nit: "affords control the" doesn't make too much sense to me. Can you change this?
</blockquote></details>

done

**Comment by jordanlewis** [comment]
LGTM

**Review by andreimatei — COMMENTED** [review]
bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/29359#-:-LXQ_lPw1LiBuZFrRlY1:b-ydi9ay)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @petermattis and @tbg)

**Pull request merged**